CMakeLists.txt

```
@@ -62,6 +62,20 @@ if(JWT_EXTERNAL_PICOJSON)
   find_package(picojson 1.3.0 REQUIRED)
 endif()
 
+find_package(nlohmann_json CONFIG)
+
+if(NOT nlohmann_json_FOUND)
+  include(FetchContent)
+
+  FetchContent_Declare(
+    json
+    GIT_REPOSITORY https://github.com/ArthurSonzogni/nlohmann_json_cmake_fetchcontent.git
+    GIT_TAG v3.9.1
```

Let's sneak a bump in :wink:

```suggestion
    GIT_TAG v3.11.2
```